tests/gui_web_e2e_test.py

```
@@ -214,18 +235,25 @@ def test_e2e_primer_lifecycle_and_state(
-    page.mouse.click(box["x"] + box["width"] - 76, box["y"] + box["height"] / 2)
+    # Click the header Delete Primer button.
+    delete_btn = page.locator("[aria-label*='Delete Primer']").first
+    if not delete_btn.is_visible():
+        # Fall back to locating the delete button in the focused row
+        # container if individual button's behaviour is merged
+        row_container = page.locator("[aria-label*='Add Primer Below']").first
+        delete_btn = row_container.locator("[role='button']").nth(1)
+    expect(delete_btn).to_be_enabled(timeout=5000)
+    try:
+        delete_btn.click(force=True)
+    except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)